tests/Dfe.EarlyYearsQualification.UnitTests/Mocks/MockContentfulServiceTests.cs

```diff
@@ -317,9 +317,10 @@ public async Task GetDatesQuestionPage_PassWhenWasQualificationStartedId_Returns
 
         result.Should().NotBeNull();
         result!.Question.Should().Be("Test Dates Questions");
-        result.CtaButtonText.Should().Be("Continue");
-        result.ErrorBannerHeading.Should().Be("There is a problem");
-        result.BackButton.Should().BeEquivalentTo(new NavigationLink
+        result!.CtaButtonText.Should().Be("Continue");
+        result!.ErrorBannerHeading.Should().Be("There is a problem");
+        result!.AwardedDateIsAfterStartedDateErrorText.Should().Be("Error- AwardedDateIsAfterStartedDateErrorText");
+        result!.BackButton.Should().BeEquivalentTo(new NavigationLink

```

Comment:
The ! isn't needed on these lines as it's already checked on the line above

```suggestion
        result.CtaButtonText.Should().Be("Continue");
        result.ErrorBannerHeading.Should().Be("There is a problem");
        result.AwardedDateIsAfterStartedDateErrorText.Should().Be("Error- AwardedDateIsAfterStartedDateErrorText");
        result.BackButton.Should().BeEquivalentTo(new NavigationLink

```